teuthology/repo_utils.py

```diff
@@ -95,7 +95,7 @@ def current_branch(path: str) -> str:
     return result
 
 
-def enforce_repo_state(repo_url, dest_path, branch, commit=None, remove_on_error=True):
+def enforce_repo_state(dest_clone, dest_path, repo_url, branch, commit=None, remove_on_error=True):
```

Comment:
I'd prefer not to change initial signature and leave repo_url at first position, and add `dest_cone` as an optional argument with defaults value.

Reply:
Sure, there will probably be lots of reworking to this code but it was a dirty hack to start because the clones after every QA suite tweak was driving me nuts. That's why it's in this PR.